### ISSUE TITLE
OCR0057: Exploration of solution for finding pecha glyph

### ISSUE THREAD

**Issue by 10kalden** [OpenPecha/pecha-glyph-generation#1]
Description:

- To create a usable Tibetan font, we required a set of glyphs, till now we have been extracting the glyphs from Pecha images using Google OCR or data in OPF.
- These methods could not extract complex ligature glyphs due to rare occurrences in the Pecha images.
- We need to think of a method to extract these glyphs or generate synthetic glyphs.

Implementation:

1. we can have the annotator manually draw the glyphs 
2. work on glyph-based font style transfer done [here](https://github.com/orgs/OpenPecha/projects/56/views/7?pane=issue&itemId=65728918)
3. we can work by creating synthetic glyph generation using diffusion model

Sub-task:

- [x] Look in the font style transfer method, there is already a base [here](https://github.com/orgs/OpenPecha/projects/56/views/7?pane=issue&itemId=65728918)
- [x] Generating glyph using diffusion model, there are some work done [here](https://github.com/orgs/OpenPecha/projects/56/views/7?pane=issue&itemId=71395401)
- [x] Prepare a report comparing the resources needed for manual annotation and model generation

Completion Criteria:
To create a method to extract or generate pecha glyphs

**Comment by 10kalden** [comment]
- I had a KT session with Norbu yesterday, his work on font style transfer for line based using GANs did not yield the result.
- Need to explore more on font style transfers on glyph based using GANs
- had a discussion to work with glyph generation using cGANS.

**Comment by 10kalden** [comment]
**Report: Conditional GAN Training for Glyph Generation**
### Project Overview
Objective: Train a conditional GAN to generate Tibetan glyphs based on two inputs:
Glyph image generated with a font.
Target Derge style glyph.
Dataset: 5,000 glyph conditions and 5,000 target Derge style images.
Model & Data Preparation
Model: Conditional GAN (cGAN) to generate glyph images.
Data: Preprocessed by resizing and normalizing paired images.
### Training Setup

GPU: 1x RTX 4070 (12 GB VRAM).
Memory: 258 GB RAM.
Cost: $0.185/hr.
Training Duration & Cost
Epoch Time: 7.5 minutes per epoch.
Cost for 40 epochs: 5 hours, $0.93.
